src/components/Progress/ProgressRing.js

```diff
@@ -106,6 +108,7 @@ ProgressRing.propTypes = {
     warning: PropTypes.string.isRequired,
     complete: PropTypes.string.isRequired,
   }),
+  children: PropTypes.node,
```

Comment:
It would make sense to document that children takes precedence over status / customIcon